Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -349,6 +349,54 @@ vault write -f transit/keys/<mybucketkey> exportable=true # transit engine
 
 * `tokenSecretName` can be (and often will be) the same for both kms and s3 configurations.
 
+## Advanced configuration
+
+!!! warning
+    This feature is intended for advanced users. It allows breaking configurations to be easily
+    applied. Rook cannot guarantee support non-recommended uses of this feature. Use with caution.
+
+CephObjectStore allows arbitrary Ceph configurations to be applied to RGW daemons that serve the
+object store. [RGW config reference](https://docs.ceph.com/en/latest/radosgw/config-ref/).
+
+Configurations are applied to all RGWs that serve the CephObjectStore. Values must be strings.
+Below is an example showing how different RGW configs and values might be applied. The example is
+intended only to show a selection of value data types.
+
+```yaml
+# THIS SAMPLE IS NOT A RECOMMENDATION
+# ...
+spec:
+  gateway:
+    # ...
+    rgwConfig:
+      debug_rgw: "20" # int
+      rgw_s3_auth_use_ldap: "true" # bool
+    rgwCommandFlags:
+      debug-rgw: "20" # config keys are allowed to be dashes or underscores
```

Comment:
The second example is specifically to show that dashes are allowed, which is why it isn't consistent and mentions dashes in the comment. I thought it would be clearer for users to see that the same configuration can be given with `-` or `_` if the keys are the same.